src/app/components/TranscriptExperiment/SignPost/index.tsx

```diff
@@ -0,0 +1,63 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { useContext, useId } from 'react';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import Text from '#app/components/Text';
+import VisuallyHiddenText from '#app/components/VisuallyHiddenText';
+import styles from './index.styles';
+import SVGs from './svgs';
```

Comment:
Let's import only the SVGs we need 

```suggestion
import { FanSvg, PlusSvg } from './svgs';
```